recipes-security/refpolicy/refpolicy-mcs-2.20141203/policy/modules/contrib/tpmutil.te

```diff
@@ -58,3 +58,19 @@ sysnet_read_config(tpmutil_t)
 statusreport_write_storage_files(tpmutil_t)
 userdom_manage_user_home_content_files(tpmutil_t)
 userdom_use_user_terminals(tpmutil_t)
+
+gen_require(`
+       type tcsd_var_lib_t, xc_config_key_t, unreserved_port_t, tpm_device_t, boot_t;
```

Comment:
The best practice is to define interfaces in the modules that own these types...
But I'm fine with this, the "SELinux way" can be painful at times...